.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ production ]
+  pull_request:
+    branches: [ production ]
```

Comment:
Does this mean this will _only_ run on push to production? That feels like an unfortunate constraint?

Reply:
Ah, it looks like it only pushes on pull requests _into_ production. Which maybe makes sense? But I'm not sure why we wouldn't want _all_ PRs / pushes to run the build?

Reply:
It will run when commit pushed or merged to production.
And yes, it will also run when pull requests into production. 

I think I want to make app pull request run the test but for push, only production should be good enough unless we have staging branch in the future.